hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -904,4 +945,7 @@ private static Map<String, String> getAuditParams(
 
   }
 
+  private boolean isExceedThreshold(long opLatencyMs) {
```

Comment:
nit: Can we change the function name to `isOperationSlow()` to make it more readable.

Reply:
Done.